Dockerfile

```diff
@@ -3,15 +3,18 @@
 ARG GO_VERSION=1.21
 ARG XX_VERSION=1.4.0
 
+# for testing
 ARG DOCKER_VERSION=25.0.2
 ARG GOTESTSUM_VERSION=v1.9.0
 ARG REGISTRY_VERSION=2.8.0
 ARG BUILDKIT_VERSION=v0.12.5
```

Comment:
Looks like we have some versions to update here (probably fine to be done separate)

Reply:
Yes done in https://github.com/docker/buildx/pull/2322